Libs/RaptureCore/src/main/java/rapture/kernel/SeriesApiImpl.java

```diff
@@ -158,6 +158,20 @@ public void deletePointsFromSeriesByPointKey(CallingContext context, String seri
         RaptureURI internalURI = new RaptureURI(seriesURI, Scheme.SERIES);
         SeriesRepo repo = getRepoOrFail(internalURI);
         repo.deletePointsFromSeriesByColumn(internalURI.getDocPath(), columns);
+        if (SearchPublisher.shouldPublish(getConfigFromCache(internalURI.getAuthority()), internalURI)) {
+            // Need to delete points from ES.
+            // Drop the old series and re-add all the points that remain.
+            // Is there a better way?
+            SearchPublisher.publishDeleteMessage(context, getConfigFromCache(internalURI.getAuthority()), internalURI);
```

Comment:
Difficult to say. It depends on the number of points in the series - how many are deleted and how many remain. If you're deleting a small number of points from a large series it's probably highly inefficient. If you're using the method to wipe out the series it's more efficient than deleting the points. We don't have a method to determine the number of points in a series without querying the entire series, which makes it difficult to judge. Also we don't have any idea how heavily used this method is likely to be.